arm-datalake-store/account/2016-11-01/swagger/account.json

```diff
@@ -0,0 +1,1311 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DataLakeStoreAccountManagementClient",
+    "description": "Creates an Azure Data Lake Store account management client.",
+    "version": "2016-11-01",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataLakeStore/accounts/{accountName}/firewallRules/{firewallRuleName}": {
```

Comment:
I agree, this is a really good idea. I will make this change, but it won't make it for the current PowerShell build in november (which code freeze is today for).